tests/api/common/experimental/mark_tasks.py

```diff
@@ -397,5 +426,153 @@ def tearDown(self):
         self.session.query(models.DagStat).delete()
         self.session.commit()
 
+
+class TestMarkTaskInstanceInDagRun(unittest.TestCase):
+    def setUp(self):
+        self.dagbag = models.DagBag(include_examples=True)
+        self.dag1 = self.dagbag.dags['test_example_bash_operator']
+        self.dag2 = self.dagbag.dags['example_subdag_operator']
+
+        self.execution_dates = [days_ago(3), days_ago(2), days_ago(1)]
+
+        self.session = Session()
+
+    def set_default_task_instance_states(self, dr):
+        if dr.dag_id != 'test_example_bash_operator':
+            return
+        # success task
+        dr.get_task_instance('runme_0').set_state(State.SUCCESS, self.session)
+        # skipped task
+        dr.get_task_instance('runme_1').set_state(State.SKIPPED, self.session)
+        # retry task
+        dr.get_task_instance('runme_2').set_state(State.UP_FOR_RETRY, self.session)
```

Comment:
Does this task have more than 1 try?

Reply:
I think it only has the default 0 set, good catch, making it to have more than 1 retries.